packages/flutter/lib/src/services/text_input.dart

```diff
@@ -930,4 +915,96 @@ class TextInput {
     }());
     return true;
   }
+
+  MethodChannel _channel;
+
+  TextInputConnection _currentConnection;
+  TextInputConfiguration _currentConfiguration;
+  TextEditingValue _currentTextEditingValue;
+
+  Future<dynamic> _handleTextInputInvocation(MethodCall methodCall) async {
+    if (_currentConnection == null)
+      return;
+    final String method = methodCall.method;
+
+    // Reattach request needs to be handled regardless of the client ID.
```

Comment:
Update this comment? Since the method is no longer called "reattach"?

Reply:
Done